### ISSUE TITLE
Fluid not working inside symbol

### ISSUE THREAD

**Issue by davetatch** [matt-curtis/Fluid-for-Sketch#134]
When I make a symbol with a text field that has auto sizing, then add text that adds extra lines, update layout doesn't take those extra lines into account. Am I doing something wrong or does fluid not support dynamically entered text?

**Comment by matt-curtis** [comment]
I'm sorry to report Fluid isn't really actively maintained. Regarding this particular issue, it's likely that some part of Sketch's text layer code changed and Fluid would need to be updated in order to address it. I plan to update the README with a notice about the lack of maintenance. I apologize for the inconvenience. There's a few alternatives to Fluid that do the same, or function similarly, that I can point you to if interested.

**Comment by pwdjean** [comment]
Yes, I have been encountering the same issues. @matt-curtis really like the tool but understand time constraints all too well. would you mind pointing us to some alternatives? Thank you!

**Comment by matt-curtis** [comment]
Here are two that I'm aware of (there are more, but I'd have to look them up.)

AnimaApp's AutoLayout plugin: https://github.com/AnimaApp/Auto-Layout
Paddy: https://github.com/DWilliames/paddy-sketch-plugin

**Comment by pwdjean** [comment]
Thanks!
